tests/ttnn/unit_tests/operations/test_ssm_prefix_scan.py

```diff
@@ -164,9 +163,7 @@ def to_device(x):
         a_chunk = to_device(a_chunks[idx])
         bx_chunk = to_device(bx_chunks[idx])
 
-        h_chunk = ttl.operations.primary.transformers.ssm_prefix_scan(
-            a_chunk, bx_chunk, h_prev, output_mem_config=memory_config, output_dtype=dtype
-        )
+        h_chunk = ttnn.experimental.ssm.prefix_scan(a_chunk, bx_chunk, h_prev, memory_config=memory_config, dtype=dtype)
```

Comment:
Should be like this
```suggestion
        h_chunk = ttnn.experimental.prefix_scan(a_chunk, bx_chunk, h_prev, memory_config=memory_config, dtype=dtype)
```

Reply:
I will leave a comment below to clarify how and why

Reply:
@ayerofieiev-tt Would it be better to make it `ttnn.experimental.ssm_prefix_scan`? 

The only reason would be because we also have `reduce_sum` and `eltwise_mul` custom ops (previously `ssm_eltwise_mul` and `ssm_1d_sum_reduce` which without the `ssm` namespace is likely to collide with something else.

Reply:
Personally, I prefer having `ttnn.transformers.<op>` or `ttnn.experimental.transformers.<op>` in python and C++, but we have decided to drop the submodule of `transformers` and `ssm`. What happens is:
- We write custom ops
- See that they are used quite often, so we consider it a generic custom op

For example, `split_query_key_and_value` op went through many iterations before we settled on what we have in `ttnn`. I would suggest do the following:
- Make the name more specific. For example, `split_query_key_and_value` is specific enough that people can probably tell it's for transformer(-esque) models
- In the documentation, explicitly describe what it's for and mention that it's for mamba, SSM's, etc...